src/mono/CMakeLists.txt

```diff
@@ -277,9 +277,9 @@ elseif(CLR_CMAKE_HOST_OS STREQUAL "windows")
   add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/wd4324>)  # 'struct_name' : structure was padded due to __declspec(align())
   add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/EHsc>)    # set exception handling behavior
   add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/FC>)      # use full pathnames in diagnostics
-  add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/GUARD:cf) # Enable control flow guard
+  append("/guard:cf" CMAKE_C_FLAGS CMAKE_CXX_FLAGS)          # Enable control flow guard
```

Comment:
the reason why it didn't work was because you were missing the closing `>`, please change it back.